src/pages/api/experimental/assetrelations/assets.md

```diff
@@ -0,0 +1,4 @@
+---
+title: Assets API (Experimental - asset relations)
+frameSrc: https://white-robin-23.redoc.ly/#tag/Relations
```

Comment:
Correct URL for public API ishttps://adobe-aem-assets-author-assetrelations.redoc.ly/#tag/Relations, currently not public though.